src/manual_folds_with_custom_loss.py

```diff
@@ -12,12 +13,15 @@
 LOG_CONSTANT = 1
 # Whether to generate parameter relationship graphs
 MAKE_GRAPHS = False
-# Which TDD option to run [0 = "off", 1 = "TDD", 2 = "log_TDD"]
-TDD_OPTION = 0
-WORKERS = 1  #-1  # set to 1 for debug mode and -1 to use all workers on your machine
+
+# Which TDD option to run [0 = "off", 1 = "TDD", 2 = "log_TDD", 3 = "1/TDD"]
+TDD_OPTION = 3
+WORKERS = -1  #-1  # set to 1 for debug mode and -1 to use all workers on your machine
 VERBOSE = False
 LOCAL_SEARCH_ON_TOP_N_RESULTS = 100
 LAST_STEP_INTERVAL = 10
+SKIP_ALREADY_RUN = False
+Y_VARIABLE_LIST = ["CIR"]  #["BASAL", "log_BASAL"]  ['BASAL', 'log_BASAL', 'ISF', 'log_ISF', 'CIR', 'log_CIR']
```

Comment:
If we're going to have this be the code for others to look at in conjunction with the paper, would we want to change `Y_VARIABLE_LIST` to reflect all the variables we ultimately ran?

Reply:
yes, I agree. I will make that change.